task/09-functions-n-closures-tasks.js

```diff
@@ -104,7 +124,20 @@ function memoize(func) {
  * retryer() => 2
  */
 function retry(func, attempts) {
-    throw new Error('Not implemented');
+    return () => {
+        for (let i = 0; i <= attempts; i++) {
```

Comment:
You dont' need 2 exit conditions here. Leave the on in the for-loop definition
![image](https://user-images.githubusercontent.com/7272208/87967815-1999c800-cac8-11ea-9aff-ce2644653858.png)